src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -587,8 +587,10 @@ export default class LazyMobXTable<T> extends React.Component<LazyMobXTableProps
 			let paginationProps:IPaginationControlsProps = {
 				className:"text-center topPagination",
 				itemsPerPage:this.store.itemsPerPage,
+                totalItems:this.store.displayData.length,
```

Comment:
I've tried to fix this indentation, for some reason when I try to change it git doesnt register it as a change and wont let me commit..